config/cesm/machines/config_machines.xml

```diff
@@ -1841,7 +1841,7 @@ This allows using a different mpirun command to launch unit tests
   </machine>
 
 
-  <machine MACH="melvin">
+  <machine MACH="mappy">
```

Comment:
Hi Jim, it looks like the Travis tests do not like the machine melvin to mappy change.

Reply:
yes, I have a fix for this I haven't pushed yet - it was actually mappy-> melvin that messed it up.